test/e2e-tests-withdraw.sh

```diff
@@ -10,7 +10,7 @@ sleep 5
 truffle exec scripts/invokeViewFunction.js 'getCurrentStateRoot'
 
 EXPECTED_HASH="77b01abfbad57cb7a1344b12709603ea3b9ad803ef5ea09814ca212748f54733"
-truffle exec scripts/invokeViewFunction.js 'getCurrentStateRoot' | grep $EXPECTED_HASH
+truffle exec scripts/invokeViewFunction.js 'getCurrentStateRoot' | grep ${EXPECTED_HASH}
```

Comment:
What’s the difference between the two?

Reply:
One is a lint warning, the other is not.